FlowCal/io.py

```diff
@@ -1095,25 +1097,29 @@ def amplifier_gain(self, channels=None):
         else:
             return self._amplifier_gain[channels]
 
-    def domain(self, channels=None):
+    def range(self, channels=None):
         """
-        Get the domain of the specified channel(s).
+        Get the range of the specified channel(s).
 
-        The domain is inferred from the $PnR parameter, as
-        ``np.arange($PnR)``. The domain should be transformed along with
-        the data when passed through a transformation function.
+        The range is inferred from the $PnR parameter as ``[0, $PnR - 1]``.
```

Comment:
ahhh ok. yeah i agree; i don't think we understand how floating point bins should be generated yet, and I think you've chosen a good default until we can develop a more appropriate model.

looks like you still have a comment which incorrectly describes range for floating point though: https://github.com/taborlab/FlowCal/blob/cee4302047983fcb9cae3edd5fd031368244471f/FlowCal/io.py#L1339